ipam/manager.go

```diff
@@ -134,7 +134,11 @@ func (am *addressManager) restore(rehydrateIpamInfoOnReboot bool) error {
 		// Check if the VM is rebooted.
 		modTime, err := am.store.GetModificationTime()
 		if err == nil {
-			rebootTime, err := platform.GetLastRebootTime()
+			p := platform.NewExecClient(nil)
+			rebootTime, err := p.GetLastRebootTime()
+			if err != nil {
```

Comment:
Do we need this check as it was not there before and we already log the error in Getlastrebootime func.?